pre_processing/read_slstr.F90

```diff
@@ -15,6 +16,29 @@ module read_slstr_m
 
    implicit none
 
+   ! The calibration of SLSTR is under ongoing revisions, such that the
+   ! application of empirical correction factors is currently required.
+   ! These are taken from the SLSTR User Guide and/or personal communications.
+   real, parameter :: slstr_correction_factor(18) = [ &
+        0.95, & ! Ch1
+        -1.0, & ! Ch2
+        -1.0, & ! Ch3
+        -1.0, & ! Ch4
+        1.10, & ! Ch5
+        1.10, & ! Ch6
+        -1.0, & ! Ch7
+        -1.0, & ! Ch8
+        -1.0, & ! Ch9
+        0.92, & ! Ch10
+        0.93, & ! Ch11
+        0.93, & ! Ch12
+        -1.0, & ! Ch13
+        -1.0, & ! Ch14
+        -1.0, & ! Ch15
+        -1.0, & ! Ch16
+        -1.0, & ! Ch17
+        -1.0 ]  ! Ch18
+
```

Comment:
Can we not just set the `-1.0` values to `1.0` and then remove the `if slstr_correction_factor(i) .gt. 0` statement in the `read_slstr_funcs.F90`